tests/main.nf.test

```diff
@@ -51,6 +51,25 @@ nextflow_pipeline {
             assert ecoli_metadata."Scheme" == "ecoli_achtman_4"
             assert ecoli_metadata."Sequence Type" == "678"
 
+            // Check the commandline parameters
```

Comment:
Should this be added to `tests/main.nf.test` or to `conf/test.config` 

```
params {
    config_profile_name        = 'Test profile'
    config_profile_description = 'Minimal test dataset to check pipeline function'

    // Limit resources so that this can run on GitHub Actions
    max_cpus   = 2
    max_memory = '6.GB'
    max_time   = '6.h'

    // Input data
    input = "https://github.com/phac-nml/staramrnf/raw/dev/assets/samplesheet.csv"
}
```

Reply:
Add to `tests/main.nf.test`.

In theory the file `tests/nextflow.config` can be used to set config options for all tests run using nf-test, but I tried briefly to set `max_memory` there and it wasn't working for me, but setting in the `tests/main.nf.test` did work.